src/main/java/com/smartbear/collaborator/issue/IssueRest.java

```diff
@@ -739,10 +762,10 @@ public String createReview(CollabUserInfo collabUserInfo) throws Exception {
 			createReviewCommand.getArgs().put("title", issue.getSummary() != null ? issue.getSummary() : "");
 			createReviewCommand.getArgs().put("customFields",
 					new com.smartbear.collaborator.json.collab.CustomField[] { 
-					new com.smartbear.collaborator.json.collab.CustomField("Overview", new HashSet(Arrays.asList(issue.getDescription() != null ? issue.getDescription() : "")) )	});
-			createReviewCommand.getArgs().put("internalCustomFields",
-					new com.smartbear.collaborator.json.collab.CustomField[] { 
-					new com.smartbear.collaborator.json.collab.CustomField("JiraIssueId",  new HashSet(Arrays.asList(issue.getId().toString())) )});
+							new com.smartbear.collaborator.json.collab.CustomField("Overview", new HashSet(Arrays.asList(issue.getDescription() != null ? issue.getDescription() : "")) ),

```

Comment:
On a related note, is there a way to access this internal custom field so I can pass it as a parameter in the Triggers? Or should `${review.custom.JiraIssueId}` still work?